src/dot_recognize/src/point_number_publisher.py

```diff
@@ -135,10 +135,10 @@ def numbercallback(self,data):
 
 if __name__ == '__main__':
     bridge=CvBridge()
```

Comment:
Did you use ros images? I see you used `cv2.VideoCapture`, therefore this line may not be necessary.